### ISSUE TITLE
Request: Mulan Open Source Community（China）

### ISSUE THREAD

**Issue by gh-Devin** [simple-icons/simple-icons#6860]
### Brand Name

Mulan Open Source Community

### Website

mulanos.cn

### Popularity Metric

650

### Official Resources for Icon and Color

svg:https://toscode.mulanos.cn/mulan-community/governance/blob/master/logos/logo-vertical.svg
color：00a0e9

### Additional Comments

Mulan Open Source Community originated in China, with MulanPSL-2.0 (https://opensource.org/licenses/MulanPSL-2.0), mainly responsible for incubating open source projects, open source governance, open source education, open source salon-related things.

### Contributing

No

**Comment by service-paradis** [comment]
Thank you @gh-Devin for your request. Unfortunately, the website is not popular enough yet to have an [Alex Rank](https://www.alexa.com/siteinfo/mulanos.cn). As of now, it looks like it does not reach our popularity requirements.

Before we close the issue, though, is there another metric we could use to assess the popularity?

**Comment by gh-Devin** [comment]
> Thank you @gh-Devin for your request. Unfortunately, the website is not popular enough yet to have an [Alex Rank](https://www.alexa.com/siteinfo/mulanos.cn). As of now, it looks like it does not reach our popularity requirements.
> 
> Before we close the issue, though, is there another metric we could use to assess the popularity?

Yes, it's true that it's not included in Alex Rank, but MulanPSL-2.0 is already used by over 100,000 code repositories in (Gitee), and the Mulan community is currently incubating 12 projects, including audio and video (SRS) https://github.com/ossrs/srs, Enterprise SCRM (LinkWechat) https://gitee.com/LinkWeChat/link-wechat?_from=gitee_search, Data Processing (Piflow) https://github.com/cas-bigdatalab/piflow, etc.

**Comment by service-paradis** [comment]
Since it does not meet our traditional popularity requirements, I will ask other @simple-icons/maintainers opinion on this on.

**Comment by gh-Devin** [comment]
> Since it does not meet our traditional popularity requirements, I will ask other @simple-icons/maintainers opinion on this on.

Okay, thank you very much!

**Comment by sachinraja** [comment]
I don't think this meets the popularity requirements. https://github.com/ossrs/srs is the only repo listed that meets our requirements (over 5k stars) and I see no indication of a connection to the Mulan community.

**Comment by service-paradis** [comment]
Closing this one for now Feel free @gh-Devin to reopen it if you have more information that cal help us assess the project popularity.
